.github/ISSUE_TEMPLATE.md

```diff
@@ -1,16 +1,17 @@
+---
+
 ### Environment summary
 
 **Install Method:** How did you install the CLI? (e.g. pip, interactive script, apt-get, Docker, MSI, nightly)  
 Answer here:
 
-**CLI Version:** What version of the CLI and modules are installed? (Available with `az --version`)  
+**CLI Version:** What version of the CLI and modules are installed? (The full output of `az --version`)  
 Answer here:
 
 **OS Version:** What OS and version are you using?  
 Answer here:
 
-**Shell Type:** What shell are you using? (e.g. bash, cmd.exe, PowerShell)  
+**Shell Type:** What shell are you using? (e.g. bash, cmd.exe, PowerShell, Bash on Windows)  
 Answer here:
 
 ### Description
```

Comment:
Since you put the separator at the top... shouldn't the description be up there?  If not, then the horizontal separator should be before description...